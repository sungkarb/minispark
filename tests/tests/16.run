./tests/16.tmp