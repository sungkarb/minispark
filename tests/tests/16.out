asdf	256
qwer	32640
